### PR TITLE
refactor(NotifyProperties): FRMW-389-convert to Quark Service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-389

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxBulkSelect.NotifyProperties` to Quark Service.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `NotifyProperties` Quark Service

### Demo Page
screen shot 2016-01-18 at 2 38 42 pm

### API Docs
screen shot 2016-01-18 at 1 38 43 pm

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/NotifyProperties/scripts/NotifyProperties.js [hunk 1]
```diff
@@ -17,19 +17,29 @@ angular.module('encore.ui.rxBulkSelect')
...
+ * <pre>
  * stats = { _numSelected: 0 };
  * scope.registerForNumSelected = NotifyProperties.registrationFn(stats, 'numSelected', '_numSelected');
- * ```
+ * </pre>
  *
- * This is saying "We have a property `_numSelected` in `stats`, and we want it exposted as `numSelected`
- * in `stats`. Whenever `stats.numSelected` is modified, other directives/controllers should be notified"
+ * This is saying "We have a property `_numSelected` in `stats`, and we want it exposed as `numSelected`
+ * in `stats`. Whenever `stats.numSelected` is modified, other directives/controllers should be notified.
```
Comment (CITguy) [inline src/quarks/NotifyProperties/scripts/NotifyProperties.js]: Please, add closing double-quote to end of line. `... notified."`

**Comment by 100stacks** [comment]
@CITguy changes made. :+1:

**Comment by parlarjb** [comment]
We don't need to change the rxBulktSelect module at all to bring this in now that it's part of quarks?

File: src/components/rxBulkSelect/rxBulkSelect.module.js [hunk 2]
```diff
@@ -15,9 +15,9 @@
  * * {@link rxBulkSelect.directive:rxBulkSelectValidate rxBulkSelectValidate}
  *
  * ## Services
- * * {@link rxBulkSelect.service:NotifyProperties NotifyProperties}
  * * {@link rxBulkSelect.service:rxBulkSelectUtils rxBulkSelectUtils}
  */
 angular.module('encore.ui.rxBulkSelect', [
+    'encore.ui.quarks',
```
Comment (100stacks) [inline src/components/rxBulkSelect/rxBulkSelect.module.js]: @parlarjb, okay injecting quarks into `rxBulkSelect`. :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**